public/index.html

```diff
@@ -311,62 +311,73 @@ <h3 class="h4 lead pt-4">
 
             <h3 class="h4 lead pt-4">GNU/Weeb Admins</h3>
             <ol>
+            
               <li>
                 <a class="a-white" href="https://t.me/nrudesu"
                   >@nrudesu</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/rival_kucheng_oren"
                   >@rival_kucheng_oren</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/ammarfaizi2"
                   >@ammarfaizi2</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/WindowZ414"
                   >@WindowZ414</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/kawaiighost"
                   >@kawaiighost</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/Ryne4S"
                   >@Ryne4S</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/lappretard"
                   >@lappretard</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/rlapz"
                   >@rlapz</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/Mysticial"
                   >@Mysticial</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/lgridsaffan"
                   >@lgridsaffan</a
                 >
               </li>
+            
               <li>
                 <a class="a-white" href="https://t.me/rin4th"
                   >@rin4th</a
                 >
               </li>
-
+            
```

Comment:
Trailing whitespace here.

Reply:
Yes, as mentioned in the commit.

The workaround is to add the `for` tags after the `<ol>` and `</li>` (`endfor`). But that'll make the source code ugly.

Reply:
```
Applying: sns: Correct `GitHub` spelling
Applying: home: Update Admin list
Applying: config: Drop unused `domain` variable
Applying: home: Use liquid `for` loop for admin list
.git/rebase-apply/patch:127: trailing whitespace.
            
.git/rebase-apply/patch:133: trailing whitespace.
            
.git/rebase-apply/patch:139: trailing whitespace.
            
.git/rebase-apply/patch:145: trailing whitespace.
            
.git/rebase-apply/patch:151: trailing whitespace.
            
warning: squelched 7 whitespace errors
warning: 12 lines add whitespace errors.
```